pkg/resources/management.cattle.io/v3/globalrole/GlobalRole.md

```diff
@@ -10,9 +10,12 @@ On create or update, the following checks take place:
 
 ### Escalation Prevention
 
-Users can only change GlobalRoles with rights less than or equal to those they currently possess. This is to prevent privilege escalation. This includes the rules in the RoleTemplates referred to in `inheritedClusterRoles`. 
+ Escalation checks are bypassed if a user has the `escalate` verb on the GlobalRole that they are attempting to update. This can also be given through a wildcard permission (i.e. the `*` verb also gives `escalate`).
```

Comment:
Nit: This applies to more than just update, Escalation checks are also present on create. I know that you didn't write this, if you could fix it I would appreciate it.